crates/task-impls/src/consensus/view_change.rs

```diff
@@ -41,8 +41,8 @@ pub(crate) async fn update_view<TYPES: NodeType, I: NodeImplementation<TYPES>>(
     consensus: Arc<RwLock<Consensus<TYPES>>>,
     cur_view: &mut TYPES::Time,
     timeout_task: &mut Option<JoinHandle<()>>,
+    double_send: bool,
```

Comment:
This was why we got duplicate view changes, it recursively broadcasted a view change in the `ViewChange` event handler in the consensus task. I'll fix the naming later.

Reply:
Hmm why do we need the double send?